formats/json/commonTest/src/kotlinx/serialization/features/inline/InlineClassesTest.kt

```diff
@@ -0,0 +1,111 @@
+@file:Suppress("INLINE_CLASSES_NOT_SUPPORTED", "SERIALIZER_NOT_FOUND")
+@file:OptIn(ExperimentalUnsignedTypes::class)
+
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.features.inline
+
+import kotlinx.serialization.*
+import kotlinx.serialization.builtins.*
+import kotlinx.serialization.encoding.*
+import kotlinx.serialization.test.*
+import kotlin.test.*
+
+@Serializable
+data class SimpleContainerForUInt(val i: UInt)
```

Comment:
These classes can be moved into `InlineClassesTest`, can't they?

Reply:
'Inline classes are only allowed on top level' says Kotlin compiler